README.md

```diff
@@ -28,6 +28,10 @@ To get a local copy up and running follow these simple example steps.
 
 ### Setup
 
+### `npm start`
+
+Runs the app in the development mode.\
+Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
 ### Install
 
```

Comment:
- [ ] Please just copying and pasting the template from Microverse is not enough to make your project look personal and professional, please change all the placeholder information to information related to you and your project and remove the parts which are not applicable for your project right now. Please kindly update your `readme file` to reflect the changes for this milestone.